docs/81-urls.md

```diff
@@ -2,6 +2,9 @@
 
 # Papi-web - Annexe technique : URLs utilisées par le serveur web
 
+> [!CAUTION]
+> Cette page décrivant les URLs utilisées par l'application est obsolète.
+
```

Comment:
I'm pretty sure I'll want to use a client mod, so I'll need accessible URLs.
I agree that the URLs will be obsolete anyway because of the reorganization, but I still think public URLs are better.
In particular, screen readers don't necessarily detect changes if the URL doesn't change.

Reply:
Not sure to understand "screen readers don't necessarily detect changes if the URL doesn't change".

Reply:
Basically this comment on Reddit: don't reinvent the wheel: https://www.reddit.com/r/htmx/comments/ti8vnj/comment/i2ukfct.
While there is a possibility to notify a screen reader of a change, the easier method of notifying a screen reader user of a change is changing the page (and the URL)